app/pages/cif/projects.tsx

```diff
@@ -20,7 +33,19 @@ export const ProjectsQuery = graphql`
       id
     }
 
-    allProjects {
+    allProjects(
+      first: $pageSize
+      offset: $offset
+      filter: {
+        projectName: { includesInsensitive: $projectName }
+        operatorByOperatorId: {
+          tradeName: { includesInsensitive: $operatorTradeName }
+        }
+        rfpNumber: { includesInsensitive: $rfpNumber }
+      }
```

Comment:
Project Status column is not searchable
```suggestion
      filter: {
        projectName: { includesInsensitive: $projectName }
        operatorByOperatorId: {
          tradeName: { includesInsensitive: $operatorTradeName }
        }
        rfpNumber: { includesInsensitive: $rfpNumber }
        projectStatusByProjectStatusId: {
          name: { includesInsensitive: $status }
        }
      }
```